en/controllers/request-response.rst

```diff
@@ -611,6 +576,68 @@ You can now use the convenience method
 :php:meth:`Cake\\Http\\Response::withLocation()` to directly set or get the
 redirect location header.
 
+Setting the Body
+----------------
+
+.. php:method:: withBody($body)
+
+To set the response body, use the `withBody()` method, which is provided by the
+:php:class:`Zend\\Diactoros\\MessageTrait`::
+
+    $response = $response->withBody($stream);
+
+    // Prior to 3.4.0 - Set the body
+    $this->response->body('My Body');
+
+Be sure that `$stream` is a :php:class:`Psr\\Http\\Message\\StreamInterface` object.
+See below on how to create a new stream.
+
+.. php:method:: withStringBody($string)
+
+In most use cases, it is simpler to set a string as the response body::
+
+    // Set a string into the body
+    $response = $response->withStringBody('My Body');
+
+    //If you want a json response
+    $response = $response->withType('application/json')
+        ->withBody(json_encode(['Foo' => 'bar']));
+
+.. versionadded:: 3.4.3
+    `withStringBody()` was added in 3.4.3
+
+You can also stream responses from files using diactoros streams::
```

Comment:
Isn't diactoros a name? Using a capital letter "Diactoros", could make it more clear for non English native speakers...